src/frontend/data/rootReducer.ts

```diff
@@ -27,6 +29,7 @@ export const rootReducer = (
   action: actionTypes,
 ) => ({
   context: context(state!.context, action),
+  player: playerContext(state!.player, action),
```

Comment:
the convention here is to name the reducer as the key to the slice of the state it's handling (eg `player` here)

Reply:
done